src/assets/styles/portfolio.css

```
@@ -0,0 +1,56 @@
+.portfolio h1 {
+  padding-top: 1rem;
+  padding-inline: 2rem;
+  padding-bottom: 0.5rem;
```

You can do padding-block with two params here. 

```suggestion
  padding-block: 1.5rem 0.5rem;
```

---

Good call!